pkgs/tools/backup/dar/default.nix

```diff
@@ -1,41 +1,62 @@
-{ lib, gccStdenv, fetchurl
-, which
-, attr, e2fsprogs
-, curl, libargon2, librsync, libthreadar
-, gpgme, libgcrypt, openssl
-, bzip2, lz4, lzo, xz, zlib
-, CoreFoundation
```

Comment:
Order is preserved. Do you mean that I add some space between the "groups"?

Reply:
No, I wanted to leave something like this variant:
```
args @ { lib, gccStdenv, fetchzip, which
  , attr, e2fsprogs # packages used only in linux
  , curl, libargon2, librsync, libthreadar # libraries
  , gpgme, libgcrypt, openssl # encryption libraries
  , bzip2, lz4, lzo, xz, zlib # compression libraries
```
Can be left without comments.

Reply:
Not really sure what exactly you mean.

I ported the multiple inputs per line to one input per line. This was on purpose since it makes files more readable and changes more composable.